alf/trainers/policy_trainer.py

```diff
@@ -196,17 +196,53 @@ def _close(self):
         """Closing operations after training. """
         pass
 
+    def _set_summary(self):
```

Comment:
_summarize_training_setting might be a better name

Reply:
updated.